<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>basicParameterSettingsWindow</class>
 <widget class="QMainWindow" name="basicParameterSettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>821</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 28pt &quot;微软雅黑&quot;;
font: 18pt &quot;Adobe Devanagari&quot;;
font: 14pt &quot;Adobe Devanagari&quot;;</string>
    </property>
    <property name="text">
     <string>1、	由于照片的差异，需要手动调整一些参数，帮助系统更好的运行。
2、	强度梯度小于最小阈值的必定不是边缘
3、	强度梯度大于最大阈值的必定是边缘
4、	扩张会连接不连续部分
5、	侵蚀会丢弃边缘部分的像素，可能会造成不连续‘
6、	请随机选择一批图片中的一张
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>选择样例图片：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBasicParameterAddress">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>631</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonBasicParameterChoice">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>280</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(177, 177, 177);</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBasicParameterImage">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>330</y>
      <width>240</width>
      <height>320</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStartAdjust">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>350</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>开始调整</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
